### ISSUE TITLE
how best to share existing tarsnap setup with Tarsnap GUI?

### ISSUE THREAD

**Issue by dch** [Tarsnap/tarsnap-gui#16]
It's not clear how best to re-use an existing tarsnap configuration yet.

This should cover:
1. cache dir permissions `chown $USER:wheel /usr/local/tarsnap-cache`
2. sharing or re-using machine key(s), perhaps regenerated with different permissions
3. sharing the `tarsnap.conf` -- atm I see no way of referring to this in the GUI so it's not clear if  /usr/local/etc/tarsnap.conf and/or ~/.tarsnaprc are being applied. I assume they will be used as usual...

**Comment by shinnok** [comment]
1. What I do now for an existing cache spec is check if it's a dir and if it is writable for the current EUID. If not the Wizard will refuse it. In the Settings page I set the LineEdit to red color to signal an error. What else could I do is a bit unclear, maybe improve some specific error cases or phrases?
2. This is again an unclear problem. Currently one can only set an existing key either via the Wizard's file browse dialog, which implies read access to it. The other case being in the Settings pane.
3. This one is handled as follows:

**Comment by dch** [comment]
thanks for the clarifications, how about I update README or some other file with more info?

**Comment by shinnok** [comment]
Sure, do a PR. README is a good choice for the time being.

**Comment by shinnok** [comment]
@dch Do you still plan to do something in regards to this? Alternatively it could be a FAQ entry in this wiki page https://github.com/Tarsnap/tarsnap-gui/wiki/Tarsnap.

I'm not sure if you can make PR for wiki pages on Github.

**Comment by shinnok** [comment]
Added a clarification as FAQ entry to https://github.com/Tarsnap/tarsnap-gui/wiki/Tarsnap. Closed.

**Comment by Folcon** [comment]
@shinnok I'm wondering if this should be highlighted to the user on startup, I spent a bunch of time trying to work out why it couldn't see my existing files.

**Comment by gperciva** [comment]
Yes, there's more that we can do here.  I'll revisit this in the next few weeks.

**Comment by Folcon** [comment]
Would be happy to go over areas which I think of as confusing in the UI if you want?

**Comment by gperciva** [comment]
Hi @Folcon!  Thanks for the offer, but not yet.  Currently I'm focusing on tests and adding doxygen comments so that I know how the code is organized.  I'm going to look at the UI once there's a stable foundation so that I don't accidentally break anything if and when I change something.

**Comment by Folcon** [comment]
@gperciva, no problem. Feel free to msg me if you want some feedback :)...
